src/json-util.cpp

```diff
@@ -559,6 +578,22 @@ void ju_get_opt_field(const nlohmann::json &j, const K &key, access_data &data,
     std::copy(bin.begin(), bin.end(), std::back_inserter(data));
 }
 
+template <typename K>
+void ju_get_opt_field(const nlohmann::json &j, const K &key, not_default_constructible<access_data> &optional,
```

Comment:
Same here. I think this be `std::optional` instead of `not_default_constructible`?

Reply:
Here neither.